.github/workflows/continuous-integration.yml

```diff
@@ -21,12 +21,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: ["alicevision/popsift-deps:cuda10.2-ubuntu18.04", "alicevision/popsift-deps:cuda11.8.0-ubuntu20.04", "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04"]
+        container: ["alicevision/popsift-deps:cuda11.8.0-ubuntu20.04", "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04"]
         build_tpe: ["Release", "Debug"]
         exclude:
         # excludes debug on this one as it has a segmentation fault during the compilation (!)
         - container: "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04"
           build_tpe: "Debug"
+        - container: "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04"
+          build_tpe: "Release"
```

Comment:
We add "alicevision/popsift-deps:cuda12.1.0-ubuntu22.04" in the list, but we disable it both for release and debug.
It also seems that it's not taken into account and only the last one is really removed from the list of build targets...